### PR TITLE
990 run 2021 simulation in python

### PR DESCRIPTION
**Pull request by MaxBetzDLR**
# Changes and Information

Please **briefly list the changes** (main added features, changed items, or corrected bugs) made:

- simulation of 2021 paper 

If need be, add additional information and what the reviewer should look out for in particular:

- Some dampings are not used in cpp and thus are commented out in python aswell

## Merge Request - Guideline Checklist

Please check our [git workflow](https://github.com/SciCompMod/memilio/wiki/git-workflow). Use the **draft** feature if the Pull Request is not yet ready to review.

### Checks by code author

- [x] Every addressed issue is linked (use the "Closes #ISSUE" keyword below)
- [x] New code adheres to [coding guidelines](https://github.com/SciCompMod/memilio/wiki/Coding-guidelines)
- [x] No large data files have been added (files should in sum not exceed 100 KB, avoid PDFs, Word docs, etc.)
- [x] Tests are added for new functionality and a local test run was successful (with and without OpenMP)
- [x] Appropriate **documentation** for new functionality has been added (Doxygen in the code and Markdown files if necessary)
- [x] Proper attention to licenses, especially no new third-party software with conflicting license has been added
- [ ] (For ABM development) Checked [benchmark results](https://github.com/SciCompMod/memilio/wiki/Agent-Based-Model-Development) and ran and posted a local test above from before and after development to ensure performance is monitored.

### Checks by code reviewer(s)

- [ ] Corresponding issue(s) is/are linked and addressed
- [ ] Code is clean of development artifacts (no deactivated or commented code lines, no debugging printouts, etc.)
- [ ] Appropriate **unit tests** have been added, CI passes, code coverage and performance is acceptable (did not decrease)
- [ ] No large data files added in the whole history of commits(files should in sum not exceed 100 KB, avoid PDFs, Word docs, etc.)
- [ ] On merge, add 2-5 lines with the changes (main added features, changed items, or corrected bugs) to the merge-commit-message. This can be taken from the **briefly-list-the-changes** above (best case) or the separate commit messages (worst case).

### CODE REVIEW

**Comment by MaxBetzDLR** [comment]
Simulation creates following gif:
simulation

**Review by HenrZu — CHANGES_REQUESTED** [review]
Thanks for the great extension :) Looks very good. A few small comments. Also, we should save & compare the graphs in cpp and py before merging again.

**Review by HenrZu — APPROVED** [review]
very good work! :)

File: pycode/examples/simulation/2021_vaccination_sarscov2_delta_germany.py [hunk 1]
```diff
@@ -0,0 +1,570 @@
...
+
+        params.DynamicNPIsInfectedSymptoms = dynamic_npis
+
+        # school holidays (holiday periods are set per node, see set_nodes)
+        school_holiday_value = mio.UncertainValue(0.5 * (1.0 + 1.0))
+        school_holiday_value.set_distribution(
+            mio.ParameterDistributionUniform(1.0, 1.0))
+        params.ContactPatterns.school_holiday_damping = mio.DampingSampling(school_holiday_value, mio.DampingLevel(InterventionLevel.Holidays.value),
+                                                                            mio:: DampingType(Intervention.SchoolClosure.value), mio.SimulationTime(0.0),
+                                                                            [Location.School.value], group_weights_all)
```
Comment (HenrZu) [inline pycode/examples/simulation/2021_vaccination_sarscov2_delta_germany.py:494]:
```suggestion
        params.ContactPatterns.school_holiday_damping = mio.DampingSampling(school_holiday_value, InterventionLevel.Holidays.value,
                                                                            Intervention.SchoolClosure.value, 0.0,
                                                                            [Location.School.value], group_weights_all)
```

**Review by HenrZu — CHANGES_REQUESTED** [review]

**Pull request merged**